GoogleForCreators/web-stories-wp
# Change the relevant color instantly when changing colors in the color picker

## Summary



## Relevant Technical Choices
Uses transform handlers to change the color of the relevant element.
Passes the necessary style rule depending on where the Color Picker is used to identify which style to apply.


## To-do

- [x] Text color
- [x] Text/shape background color

- [x] Overlay color

- [x] Border color -- that's a bit tricky since the border is applied via pseudo element. Need to check if perhaps CSS variables could help here



## User-facing changes



## Testing Instructions
I
- Add a text element
- Change the color and the background color in Fill and Highlight mode
- Verify that the relevant color changes instantly when dragging the cursor around in the Color Picker

II Test the same for Shape and background color

III Test the same for Page background color

IV Test the same for Background media overlay colors

V Test the same for border color -- please test all: Inside, outside and center border separately.

VI Verify the same for text in edit mode.



---



Fixes #4463
